db/schema.rb

```diff
@@ -472,7 +472,6 @@
     t.datetime "updated_at", null: false
     t.bigint "user_id"
     t.datetime "verified_at"
-    t.index ["user_id"], name: "index_email_authorizations_on_user_id"
```

Comment:
This keeps popping up for me locally, does anyone know anything about it?

Reply:
I had this issue and updated my database manually via a console to get it to go away. I always figure it came from a branch I tested that never ended up in master

Reply:
Yes, that fixed it. I just tried to avoid a similar situation to what we had a few months back were different PRs alternatively added and then dropped a specific index. 😆